spyder/dependencies.py

```diff
@@ -8,10 +8,128 @@
+    {'modname': "pylint",
+     'package_name': "pylint",
+     'features': _("Static code analysis"),
+     'required_version': PYLINT_REQVER},
+    {'modname': "psutil",
+     'package_name': "psutil",
+     'features': _("CPU and memory usage info in the status bar"),
+     'required_version': PSUTIL_REQVER},
+    {'modname': "qtawesome",
+     'package_name': "qtawesome",
+     'features': _("To have an icon theme based on FontAwesome."),
+     'required_version': QTAWESOME_REQVER},
+    {'modname': "qtpy",
+     'package_name': "qtpy",
+     'features': _("Abstraction layer for Python Qt bindings so that Spyder can run on multiple Qt bindings and versions."),
+     'required_version': QTPY_REQVER},
+    {'modname': "pickleshare",
+     'package_name': "pickleshare",
+     'features': _("Show import completions on the Python consoles."),
+     'required_version': PICKLESHARE_REQVER},
+    {'modname': "zmq",
+     'package_name': "pyzmq",
+     'features': _("Client for the language server protocol (LSP)."),
+     'required_version': PYZMQ_REQVER},
+    {'modname': "chardet",
+     'package_name': "chardet",
+     'features': _("Character encoding auto-detection in Python."),
+     'required_version': CHARDET_REQVER},
+    {'modname': "numpydoc",
+     'package_name': "numpydoc",
+     'features': _("Used by Jedi to get function return types from Numpydocstrings."),
+     'required_version': NUMPYDOC_REQVER},
+    {'modname': "spyder_kernels",
+     'package_name': "spyder-kernels",
+     'features': _("Jupyter kernels for the Spyder console."),
+     'required_version': SPYDER_KERNELS_REQVER},
+    {'modname': "qdarkstyle",
+     'package_name': "qdarkstyle",
+     'features': _("Dark style for the entire interface"),
+     'required_version': QDARKSTYLE_REQVER},
+    {'modname': "atomicwrites",
+     'package_name': "atomicwrites",
+     'features': _("Atomic file writes."),
+     'required_version': ATOMICWRITES_REQVER},
+    {'modname': "diff_match_patch",
+     'package_name': "diff_match_patch",
+     'features': _("Compute text file diff changes during edition."),
+     'required_version': DIFF_MATCH_PATCH_REQVER},
+    {'modname': "watchdog",
+     'package_name': "watchdog",
+     'features': _("Watch file changes on project directories."),
+     'required_version': WATCHDOG_REQVER},
+    {'modname': "keyring",
+     'package_name': "keyring",
+     'features': _("Save Github credentials to report errors securely."),
+     'required_version': KEYRING_REQVER},
+    {'modname': "pexpect",
+     'package_name': "pexpect",
+     'features': _("Connect to remote kernels through SSH."),
+     'required_version': PEXPECT_REQVER},
+    {'modname': "pympler",
+     'package_name': "pympler",
+     'features': _("Development tool to measure, monitor and analyze the memory behavior of Python objects in a running Python application."),
+     'required_version': PYMPLER_REQVER}]
+
+if sys.platform == 'Linux':

```

Comment:
Please change this to

    if sys.platform.startswith('linux'):

That's the way we check for Linux in our codebase.